docs/en/sql-reference/sql-functions/string-functions/crc32.md

```diff
@@ -29,9 +29,16 @@ MySQL [(none)]> select crc32(null);
 +-------------+
 | crc32(NULL) |
 +-------------+
-|        NULL |
+|   558161692 |
```

Comment:
https://dev.mysql.com/doc/refman/8.0/en/mathematical-functions.html#function_crc32
when the input is null, the result should be NULL

Reply:
crc32 is the hash value of our sub-bucket. It might make more sense to follow the internal implementation of our system.